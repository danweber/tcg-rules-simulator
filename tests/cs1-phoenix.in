VERSION:2
# [On Deletion] Play this card without paying its cost.
FIELD1:Phoenix,CS1-Phoenix,REST
FIELD2:Attack1,DW1-Van Attack2,DW1-Van Blue,CS2-02
HAND2:CS2-Quick
GAME:P2 T2 MAIN 7
DUMP
ECHO: -- Attack, Phoenix rerises
CMD:ATTACK Attack1 Phoenix
DUMP
ECHO: -- delete by option, it rerises
CMD:PLAY CS2-Quick
STEP:AUTOANSWER:CAPTURE
DUMP
