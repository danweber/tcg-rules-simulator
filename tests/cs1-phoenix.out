FIELD1:Phoenix,CS1-Phoenixdefender,REST
FIELD2:Attack1,DW1-Vanillamon Attack2,DW1-Vanillamon Blue,CS2-Sharpblade
GAME:P2 T2 MAIN 7
ECHO: -- Attack, Phoenix rerises
FIELD1:I5,CS1-Phoenixdefender
FIELD2:Attack1,DW1-Vanillamon,REST Attack2,DW1-Vanillamon Blue,CS2-Sharpblade
GAME:P2 T2 MAIN 7
ECHO: -- delete by option, it rerises
FIELD1:I6,CS1-Phoenixdefender
FIELD2:Attack1,DW1-Vanillamon,REST Attack2,DW1-Vanillamon Blue,CS2-Sharpblade
GAME:P2 T2 MAIN 1
